.github/workflows/go.yml

```diff
@@ -8,24 +8,20 @@ on:
 permissions:
   contents: read
 
-env:
-  # Minimum supported Go toolchain
-  ACTION_MINIMUM_TOOLCHAIN: "1.12.x"
-
 jobs:
   build:
     name: "Build"
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ['1.17.x', '1.18.x', '1.19.x', '1.20.x']
+        go: ['1.12.x', '1.24.x', '1.23.x']
```

Comment:
This is not in a chronological order which seems confusing.

Reply:
fixed